pkg/pf/tfbridge/provider.go

```diff
@@ -316,13 +319,56 @@ func (p *provider) terraformDatasourceName(functionToken tokens.ModuleMember) (s
 	return "", fmt.Errorf("[pf/tfbridge] unknown datasource token: %v", functionToken)
 }
 
-// NOT IMPLEMENTED: Call dynamically executes a method in the provider associated with a component resource.
-func (p *provider) CallWithContext(_ context.Context,
+func (p *provider) returnTerraformConfig() (resource.PropertyMap, error) {
+	// Get the current configuration
+	config, err := convert.EncodePropertyMapToDynamic(p.configEncoder, p.configType, p.lastKnownProviderConfig)
+	if err != nil {
+		return nil, fmt.Errorf("error encoding property map: %v", err)
+	}
+	tfConfigValue, err := config.Unmarshal(p.configType)
+	if err != nil {
+		return nil, fmt.Errorf("error unmarshaling protov6.DynamicValue: %v", err)
+	}
+
+	if !tfConfigValue.IsFullyKnown() {
+		return nil, fmt.Errorf("cannot use terraformConfig that contains unknown tftype values")
```

Comment:
Good place to instruct the user what she should do with this, or where to find help.